models/DataObject/ClassDefinition/Data/UrlSlug.php

```diff
@@ -473,7 +463,7 @@ public function setDomainLabelWidth(?int $domainLabelWidth): static
         return $this;
     }
 
-    public function preGetData(mixed $container, array $params = []): mixed
+    public function preGetData(mixed $container, array $params = []): array
```

Comment:
Thank you for the PR ❤️ !

Just noticed that this change itself is fine but any class extending `UrlSlug` could be affected by a BC-break because they may be not covariant of this `array` anymore due to the signature change (mixed > array > mixed)

Reply:
Ok let's change it back

```suggestion
    public function preGetData(mixed $container, array $params = []): mixed
```